### ISSUE TITLE
Create sreenshots for iPhone

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/maker-starter-app#99]
Guideline 2.3.3 - Performance - Accurate Metadata

We noticed that your screenshots do not sufficiently show your app in use. Specifically, your 12.9-inch iPad Pro, 6.5-inch iPhone, and 5.5-inch iPhone screenshots.

To help users understand your app’s functionality and value, your screenshots should highlight your app's core concept. For example, a gaming app should feature screenshots that capture actual gameplay within the app.

Next Steps

Please revise your screenshots to ensure that they accurately reflect the app in use on the supported devices.

Keep in mind the following requirements:

- Marketing or promotional materials that do not reflect the UI of the app are not appropriate for screenshots.
- The majority of the screenshots should highlight your app's main features and functionality.
- Confirm that your app looks and behaves identically in all languages and on all supported devices.
- Make sure that the screenshots show your app in use on the correct device. For example, iPhone screenshots should be taken on iPhone, not on iPad.

Resources

- To learn more about creating great screenshots for the App Store, see [Optimizing Your App Store Product Page](https://developer.apple.com/app-store/product-page/).
- To learn more about uploading screenshots, see [App Store Connect Help](https://help.apple.com/app-store-connect/#/devd1093d90d).

**Comment by Andrewsopoku** [comment]
I have taken some screenshots. Should i share them here?

**Comment by marvinmarnold** [comment]
@Andrewsopoku yes please FYI, these are the ones we are using for iPad, so iPhone should be similar
![IMG_0498](https://user-images.githubusercontent.com/1311030/161760320-d8d494e2-6c4e-4e90-a730-c346a59b2424.PNG)
![IMG_0499](https://user-images.githubusercontent.com/1311030/161760324-6692c4f1-6425-4d6d-bd9a-f67aab100ba3.PNG)
![IMG_0500](https://user-images.githubusercontent.com/1311030/161760326-187c3573-93f4-47ed-b1e9-d7c4203d233c.PNG)
![IMG_0501](https://user-images.githubusercontent.com/1311030/161760330-e060d0e8-c06c-47d3-967e-fff6f20ddc34.PNG)
![IMG_0502](https://user-images.githubusercontent.com/1311030/161760331-b4b05b74-0770-4225-b4e1-2a9155999243.PNG)
![IMG_0503](https://user-images.githubusercontent.com/1311030/161760332-274593a1-80bb-44f8-84be-72fbb5ac569a.PNG)

**Comment by Andrewsopoku** [comment]
![IMG_0728](https://user-images.githubusercontent.com/20680743/161777740-6dfc510f-f1b4-4218-8519-d957a3f9467f.PNG)
![IMG_0729](https://user-images.githubusercontent.com/20680743/161777749-65f428d7-2362-42d6-856d-7d7a81a1ad80.PNG)
![IMG_0732](https://user-images.githubusercontent.com/20680743/161777755-ec3f976b-93fb-44a8-84f1-50327e53f354.PNG)
![IMG_0733](https://user-images.githubusercontent.com/20680743/161777761-0617c2ff-dfd6-48cb-b720-6b352351ef99.PNG)
![IMG_0734](https://user-images.githubusercontent.com/20680743/161777778-df4610c4-ec2a-4f63-a650-f472ef84adcf.PNG)
![IMG_0735](https://user-images.githubusercontent.com/20680743/161777782-45702bab-4af8-47c5-a869-1d211ebd3508.PNG)

**Comment by marvinmarnold** [comment]
@Andrewsopoku what size are these for?

**Comment by Andrewsopoku** [comment]
@marvinmarnold from iphone 13 Pro so i guess  5.78 X 2.82 inches

**Comment by marvinmarnold** [comment]
Actually it is only accepting them as a 6.5", but that's fine.

**Comment by Andrewsopoku** [comment]
![Simulator Screen Shot - iPhone 8 Plus - 2022-04-08 at 13 22 29](https://user-images.githubusercontent.com/20680743/162435167-8e8b6d9b-afea-49fa-98ec-b04e67e48cb1.png)
![Simulator Screen Shot - iPhone 8 Plus - 2022-04-08 at 13 22 40](https://user-images.githubusercontent.com/20680743/162435200-98591f28-4b85-4ad7-a0f5-20e360d91111.png)
![Simulator Screen Shot - iPhone 8 Plus - 2022-04-08 at 13 23 04](https://user-imag
![Simulator Screen Shot - iPhone 8 Plus - 2022-04-08 at 13 23 15](https://user-images.githubusercontent.com/20680743/162435218-cb3ff5bf-52ea-431d-ae36-0e55d764d76d.png)
es.githubusercontent.com/20680743/162435209-133b2022-b3e2-4b5e-b13a-6d1c9b8540f4.png)
![Simulator Screen Shot - iPhone 8 Plus - 2022-04-08 at 13 25 06](https://user-images.githubusercontent.com/20680743/162435230-b65a0edf-dd45-4986-9f46-cc8cfee03946.png)
